test/main_test.js

```diff
@@ -0,0 +1,161 @@
+const expect = require('chai').expect;
+const WebSocket = require('ws');
+const server = require('../app.js')
+const client_util = require('../util/client_util').client_util
+let roomId = ''
+describe('room', () => {
+
+  it('Test create room', (done) => {
+    var ws = new WebSocket('ws://localhost:3000');
+    ws.onopen = function () {
+      console.log('client 1 is open')
+    }
+    ws.onmessage = function (e) {
+      jsonObj = JSON.parse(e.data);
+      switch (jsonObj.type) {
+        case 'register':
+          expect(jsonObj.playerId).to.be.string
```

Comment:
do you need a check for what value the playerId is (eg. expect(jsonObj.playerId).equal("Ahmed") or something)?

Reply:
Unfortunately playerId is a unique id